test/integration/cli.spec.js

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs'
+import test from 'tape'
+import { exec } from 'child_process'
+
+let PROJECT_KEY
+if (process.env.CI === 'true')
+  PROJECT_KEY = process.env.SPHERE_PROJECT_KEY
+else
+  PROJECT_KEY = process.env.npm_config_projectkey
+
+// Since the CLI keeps listening to stdin we need to close it manually
+test.onFinish(() => process.exit(0))
+
+test('CLI help flag', (t) => {
+  exec('csvparserprice --help', (error, stdout, stderr) => {
+    t.true(String(stdout).match(/help/), 'outputs help text')
+    t.false(error && stderr, 'returns no error')
+    t.end()
+  })
+})
+
+test('CLI takes input from file', (t) => {
+  const csvFilePath = './test/helpers/simple-sample.csv'
+
+  exec(`csvparserprice -p ${PROJECT_KEY} --inputFile ${csvFilePath}`,
+    (error, stdout, stderr) => {
+      t.true(stdout.match(/prices/), 'outputs data including \'prices\'')
+      t.false(error && stderr, 'returns no error')
+      t.end()
+    })
+})
+
+test('CLI writes output to file', (t) => {
+  const csvFilePath = './test/helpers/simple-sample.csv'
+  const jsonFilePath = '/tmp/output.json'
```

Comment:
we should clear this after test, you can use [this](https://www.npmjs.com/package/tmp) or any other module for that